ipatests/pytest_plugins/integration/tasks.py

```diff
@@ -1381,3 +1381,29 @@ def add_dns_zone(master, zone, skip_overlap_check=False,
                                     host.hostname + ".", '--a-rec', host.ip])
     else:
         logger.debug('Zone %s already added.', zone)
+
+
+def run_dnsserver_find(host, server_to_find, returncode=None):
+    """
+    Find DNS-server details
+    """
+    kinit_admin(host)
+    cmd = host.run_command(["ipa", "dnsserver-find", server_to_find],
+                           raiseonerr=False)
+    if cmd.returncode == returncode:
+        print cmd.stdout_text
```

Comment:
You cannot use print statement because it's not compatible with Python 3. If you **have** to use print, then you must add ``from __future__ import print_function`` to the top of the file and use the print **function**. In general it's a bad idea to print in tests.